backend/src/main/java/com/example/skicurort/equipment/EquipmentService.java

```diff
@@ -1,20 +1,65 @@
 package com.example.skicurort.equipment;
 
+import static com.example.skicurort.equipment.EquipmentMapper.mapToDTO;
+import static com.example.skicurort.equipment.EquipmentMapper.mapToDTOs;
+import static com.example.skicurort.equipment.EquipmentMapper.mapToEntity;
+
 import java.util.List;
+import java.util.NoSuchElementException;
+import lombok.RequiredArgsConstructor;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
 
 @Service
+@RequiredArgsConstructor
 public class EquipmentService {
 
   private EquipmentRepository equipmentRepository;
 
   @Autowired
-  public EquipmentService(EquipmentRepository equipmentRepository) {
+  EquipmentService(EquipmentRepository equipmentRepository) {
```

Comment:
konstruktor zrobi lombok - tylko musisz albo zrobić to pole final, żeby `@RequiredArgsConstructor` złapał, albo użyć `@AllArgsConstructor`